lib/summon/ris_mappings.rb

```diff
@@ -43,7 +43,6 @@
   'SN  -': ->() { issns.empty? ? isbns : issns }, #ISSN/ISBN
   'SP  -': ->() { start_page }, #Start Page
   'T1  -': ->() { subtitle ? "#{title}: #{subtitle}" : title}, #Primary Title
-  'T1  -': ->() { subtitle ? "#{title}: #{subtitle}" : title}, #Primary Title
```

Comment:
This looks fine (a duplicate line) but also is another line of actual code change in PR labeled as only a test change